CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## [Unreleased]
 
+- Add a one-click development setup using Gitpod
+
```

Comment:
This isn’t going to make any difference to end users of the package so shouldn’t be in the CHANGELOG.

```suggestion

```